cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -535,6 +539,7 @@ readableChainDBTracer tracer = Tracer $ \case
     ChainDB.StreamFromVolDB _ _ _ -> tr $ WithTip tip "StreamFromVolDB"
     _ -> pure ()  -- TODO add more iterator events
   WithTip tip (ChainDB.TraceImmDBEvent _ev) -> tr $ WithTip tip "TraceImmDBEvent"
+  WithTip tip (ChainDB.TraceVolDBEvent _ev) -> tr $ WithTip tip "TraceVolDBEvent"
```

Comment:
(Not necessarily for this PR) Can we also pattern-match on the wrapped event and trace them too? Because now sometimes we get a whole bunch of `TraceImmDBEvent` events without any message after it. Same for `TraceVolDBEvent`.

Reply:
https://github.com/input-output-hk/cardano-node/issues/576